### ISSUE TITLE
last_batch

### ISSUE THREAD

**Issue by wb1997-cell** [ZhaoZhibin/UDTL#9]
When using MMD, the 50th epoch will be wrong, because the dimension of Batch is not matched, your parser  

parser.add_argument('--last_batch', type=bool, default=False, help='whether using the last batch')
  drop_last=(True if args.last_batch and x.split('_')[1] == 'train' else False))
The default should be True

**Comment by ZhaoZhibin** [comment]
> 

Yes, you are right. When using distance-based methods, we should drop the final batch. But when using other methods,  we do not need to drop the final batch

**Comment by wb1997-cell** [comment]
thanks
